mps_youtube/mpris.py

```diff
@@ -92,7 +92,10 @@ def listenstatus(self, conn):
         """
         try:
             while True:
-                data = conn.recv()
+                try:
```

Comment:
there is a race condition. when KeyboardInterrupt is received while handling ` if isinstance(data, tuple):` section, it will still kill the mpris process. Try/catch needs to deal with those too.

I would be nice to avoid nesting try/catch statements. Redesign the existing try/catch inside the loop and just add another except.

Reply:
Yep, good idea. I pushed some changes.